locales/en.json

```diff
@@ -1,19 +1,19 @@
 {
   "urls": {
-    "battery_storage_installation": "https://www.rewiringamerica.org/app/ira-calculator/information/battery-storage-installation",
-    "efficiency_rebates": "https://www.rewiringamerica.org/app/ira-calculator/information/whole-home-energy-reduction-rebates",
-    "electric_panel": "https://www.rewiringamerica.org/app/ira-calculator/information/electrical-panel",
-    "electric_stove": "https://www.rewiringamerica.org/app/ira-calculator/information/electric-stove-induction-stove",
-    "electric_vehicle_charger": "https://www.rewiringamerica.org/app/ira-calculator/information/ev-charger",
-    "electric_wiring": "https://www.rewiringamerica.org/app/ira-calculator/information/electrical-wiring",
-    "geothermal_heating_installation": "https://www.rewiringamerica.org/app/ira-calculator/information/geothermal-heating-installation",
-    "heat_pump_air_conditioner_heater": "https://www.rewiringamerica.org/app/ira-calculator/information/heat-pump-air-conditioner-heater",
-    "heat_pump_clothes_dryer": "https://www.rewiringamerica.org/app/ira-calculator/information/heat-pump-clothes-dryer",
-    "heat_pump_water_heater": "https://www.rewiringamerica.org/app/ira-calculator/information/heat-pump-water-heater",
-    "new_electric_vehicle": "https://www.rewiringamerica.org/app/ira-calculator/information/electric-vehicles",
-    "rooftop_solar_installation": "https://www.rewiringamerica.org/app/ira-calculator/information/rooftop-solar-installation",
-    "used_electric_vehicle": "https://www.rewiringamerica.org/app/ira-calculator/information/electric-vehicles",
-    "weatherization": "https://www.rewiringamerica.org/app/ira-calculator/information/weatherization"
+    "battery_storage_installation": "https://homes.rewiringamerica.org/federal-incentives/25d-battery-storage-tax-credit",
```

Comment:
I think these make more sense as URLs on the IRA programs, not on items. You'd add them in `src/data/programs.ts`. Then you'll also have to edit `src/routes/v0.ts` to populate `more_info_url` with the program URL, rather than the item URL.

Then these item URLs would become sort of useless. They wouldn't appear at all in v0. They'd still be there in v1, but the frontend doesn't use them except as a fallback if there's no `program_url` (and I think all non-IRA programs have URLs now). As a followup, we should make `url` required on Program objects, make `program_url` required in v1, and reflect that in the frontend code.  And possibly get rid of these item URLs altogether?